src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExperimentalOptions.cs

```diff
@@ -43,6 +43,7 @@ public ExperimentalOptions(IConfiguration configuration)
 
     /// <summary>
     /// Gets a value indicating whether retries should be enabled in case of transient errors.
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#retry
```

Comment:
nit: IMO this should go in remarks. Something like...

```csharp
/// <summary>
/// Gets a value indicating whether or not in-memory retry should be enabled for transient errors.
/// </summary>
/// <remarks>
/// Specification: <see
/// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#retry"/>.
/// </remarks>
```

Reply:
done. Thanks!